orc8r/cloud/go/services/certifier/obsidian/handlers/handlers.go

```diff
@@ -177,7 +177,41 @@ func loginHandler(c echo.Context) error {
 	if err != nil {
 		return obsidian.MakeHTTPError(err, http.StatusInternalServerError)
 	}
-	return c.JSON(http.StatusOK, res.PolicyLists)
+	return c.JSON(http.StatusOK, policyListProtoToModel(res.PolicyLists))
+}
+
+func policyListProtoToModel(policyLists []*protos.PolicyList) interface{} {
```

Comment:
1. Break this fn down into (a) the part that loops over each policy, and (b) a call into a sub-function that convernts a single policy proto to a single policy swagger object
2. Make the return type the swagger type, rather than an iface

Reply:
Ok done, also just moved all these protos to model fns to a helper file, thought it was cleaner